Documentation/conformance-tests.md

```diff
@@ -2,41 +2,52 @@
 
 The conformance test checks that a kubernetes installation supports a minimum required feature set.
 
-## Kubernetes Cluster
+## Prerequisites
 
-You will need to have a cluster already running, and a kubeconfig with the current-context set to the cluster you wish to test.
+You will need to have a Kubernetes cluster already running, and a kubeconfig with the current-context set to the cluster you wish to test.
 
 Make sure only the essential pods are running, and there are no failed or pending pods. If you are testing a small / development cluster, you may need to increase the node memory or tests could inconsistently fail (e.g. 2048mb for single-node vagrant).
 
-## Clone Kubernetes
+
+## Running the Tests
+
+Follow these steps to run the conformance test against the desired Kubernetes release:
+
+### Clone Kubernetes
+
+First, clone the Kubernetes codebase:
 
 ```sh
 $ git clone https://github.com/kubernetes/kubernetes.git
 ```
 
-## Checkout Conformance Test branch
+### Checkout Branch
+
+Next, checkout the branch or release you'd like to test against:
 
 ```sh
 $ cd kubernetes
-$ git checkout conformance-test-v1
+$ git checkout release-1.0
```

Comment:
It's better to provide specific version, IMO, then generic vX.Y.Z. Your phrase `checkout the branch or release you'd like to test against` already specifies, that version can be changed.

I would suggest to use specific release tag, so that user will see that this is Kubernetes version, like `v1.1.8` in other places in documentation (since user usually already knows which version of Kubernetes he is using).